src/test/e2e/drawer-is-visible.spec.ts

```diff
@@ -1,7 +1,10 @@
 import { test, expect, Page } from '@playwright/test';
+import { t } from 'i18next';
 
-const labelButtonToOpenDrawer: string = 'Button to open the drawer';
-const labelButtonToExitDrawer: string = 'Button to exit the drawer';
+import '../../../i18n';
```

Comment:
this looks like a hardcode, what are you importing? this looks strange, need to find another option

Reply:
Yes, I agree. I tried to find a solution, but unfortunately, if I pass this path to the global-setup.ts function for playwright, localization is not applied. I will try to find a better solution.